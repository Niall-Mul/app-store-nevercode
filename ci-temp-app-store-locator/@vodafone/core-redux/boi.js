/**
 * Copyright 2015-present Vodafone. All Rights Reserved.
 *
 * @flow
 */

// $FlowFixMe
import Config from './config';

import { AuthenticationActions } from './actions/boi/AuthenticationActions';
import { ConnxUserActions } from './actions/boi/ConnxUserActions';
import {
  InsuranceQuoteActions,
  QuoteTypes,
} from './actions/boi/InsuranceQuoteActions';

import {
  User,
  AuthenticationStatus,
} from './constants/boi/AuthenticationImmutables';
import {
  PersonalDetails,
  UserPreferences,
} from './constants/boi/UserPreferencesImmutables';
import {
  Quote,
  QuoteResponse,
  QuoteResults,
} from './constants/boi/InsuranceQuoteImmutables';

import AuthenticationReducer from './reducers/AuthenticationReducer';
import UserPreferencesReducer from './reducers/UserPreferencesReducer';
import InsuranceQuoteReducer from './reducers/InsuranceQuoteReducer';

export {
  AuthenticationActions,
  ConnxUserActions,
  InsuranceQuoteActions,
  QuoteTypes,
  User,
  AuthenticationStatus,
  PersonalDetails,
  UserPreferences,
  Quote,
  QuoteResponse,
  QuoteResults,
  AuthenticationReducer,
  UserPreferencesReducer,
  InsuranceQuoteReducer,
};

export default Config;
