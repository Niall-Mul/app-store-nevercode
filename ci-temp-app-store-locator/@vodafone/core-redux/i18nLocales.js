/**
 * Copyright 2018-present Singlepoint. All Rights Reserved.
 *
 * @flow
 */

import i18nLocales from './i18nLocales/i18nLocales.js';

export default i18nLocales;
