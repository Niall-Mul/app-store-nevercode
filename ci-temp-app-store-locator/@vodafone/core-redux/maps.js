/**
 * Copyright 2018-present Singlepoint. All Rights Reserved.
 *
 * @flow
 */

// $FlowFixMe
import Config from './config';

import { NearbyStoreActions } from './actions/NearbyStoreActions';
import {
  StoreRecord,
  NearbyStoresRecord,
} from './constants/NearbyStoreImmutables';
import NearbyStoresReducer from './reducers/NearbyStoreReducer';

import { GoogleMapActions } from './actions/GoogleMapActions';
import {
  PositionRecord,
  GoogleMapRecord,
  DefaultRegion,
  RegionRecord,
} from './constants/GoogleMapImmutables';
import GoogleMapReducer from './reducers/GoogleMapReducer';

import { GoogleMapAutocompleteActions } from './actions/GoogleMapAutocompleteActions';
import {
  AutoCompleteQuery,
  SuggestionDetailsQuery,
  AutocompleteSuggestionRecord,
  AutocompleteSuggestionsRecord,
  SuggestionDetailsRecord,
  SuggestionDetailsContainerRecord,
} from './constants/GoogleMapAutocompleteImmutables';
import GoogleMapAutocompleteReducer from './reducers/GoogleMapAutocompleteReducer';
import GoogleMapAutocompleteQueryReducer from './reducers/GoogleMapAutocompleteQueryReducer';
import GoogleMapSuggestionDetailsReducer from './reducers/GoogleMapSuggestionDetailsReducer';
import GoogleMapChangeViewpointReducer from './reducers/GoogleMapChangeViewpointReducer';
import { GoogleMapDirectionActions } from './actions/GoogleMapDirectionActions';
import {
  OriginDestinationRequest,
  GoogleMapDirectionsRecord,
} from './constants/GoogleMapDirectionsImmutables';
import GoogleMapDirectionsReducer from './reducers/GoogleMapDirectionsReducer';
import { GoogleMapDistanceActions } from './actions/GoogleMapDistanceActions';

export {
  NearbyStoreActions,
  StoreRecord,
  NearbyStoresRecord,
  NearbyStoresReducer,
  GoogleMapActions,
  PositionRecord,
  GoogleMapRecord,
  DefaultRegion,
  RegionRecord,
  GoogleMapReducer,
  GoogleMapAutocompleteActions,
  AutoCompleteQuery,
  SuggestionDetailsQuery,
  AutocompleteSuggestionRecord,
  AutocompleteSuggestionsRecord,
  SuggestionDetailsRecord,
  SuggestionDetailsContainerRecord,
  GoogleMapAutocompleteReducer,
  GoogleMapAutocompleteQueryReducer,
  GoogleMapSuggestionDetailsReducer,
  GoogleMapChangeViewpointReducer,
  GoogleMapDirectionActions,
  OriginDestinationRequest,
  GoogleMapDirectionsRecord,
  GoogleMapDirectionsReducer,
  GoogleMapDistanceActions,
};

export default Config;
