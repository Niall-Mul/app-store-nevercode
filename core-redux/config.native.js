const Config = {
  REACT_APP_API_ENDPOINT: 'https://api.vodafone.cloud.singlepoint.ie/',

  // Note that Google Maps is no longer free and keys below are Singlepoint keys
  REACT_APP_GOOGLE_PLACES_API_KEY: 'AIzaSyBZEBNkNXEI5zv3_CJ8bj37bx3-AhTVwbU',
  REACT_APP_GOOGLE_DIRECTIONS_API_KEY:
    'AIzaSyCe6dfcGeiztbxXTuyh1Fe6atiGU6gDqeo',
};
export default Config;
