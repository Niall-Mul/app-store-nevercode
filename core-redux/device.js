/**
 * Copyright 2018-present Singlepoint. All Rights Reserved.
 *
 * @flow
 */

// $FlowFixMe
import Config from './config';

import { DeviceActions } from './actions/DeviceActions';
import { DeviceRecord } from './constants/DeviceImmutables';
import DeviceReducer from './reducers/DeviceReducer';

export { DeviceActions, DeviceRecord, DeviceReducer };

export default Config;
