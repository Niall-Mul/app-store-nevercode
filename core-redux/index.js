/**
 * Copyright 2018-present Singlepoint. All Rights Reserved.
 *
 * @flow
 */

// $FlowFixMe
import Config from './config';

import { CounterActions } from './actions/CounterActions';
import { CounterRecord } from './constants/CounterImmutables';
import CounterReducer from './reducers/CounterReducer';

import { HelloApiActions } from './actions/HelloApiActions';
import {
  HelloApiRecord,
  HelloApiResults,
} from './constants/HelloApiImmutables';
import HelloApiReducer from './reducers/HelloApiReducer';

export {
  CounterActions,
  CounterRecord,
  CounterReducer,
  HelloApiActions,
  HelloApiRecord,
  HelloApiResults,
  HelloApiReducer,
};

export default Config;
