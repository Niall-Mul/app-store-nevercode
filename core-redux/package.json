{
  "name": "@vodafone/core-redux",
  "version": "0.0.1",
  "description": "Vodafone common Redux Actions/Reducers and Libraries",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: run tests at this level via your native or web app\" && exit 1"
  }
}
