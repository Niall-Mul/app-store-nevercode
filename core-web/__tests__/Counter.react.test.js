import React from 'react';
import ReactDOM from 'react-dom';
import ReactTestUtils from 'react-dom/test-utils';
import renderer from 'react-test-renderer';
import Counter from '../components/Counter.react';
import { CounterRecord } from '@vodafone/core-redux';

describe('Counter Component', () => {
  let root;
  let props;
  let mountedBurgerMenu;

  it('renders without crashing', () => {
    const div = document.createElement('div');
    const counterRecord = new CounterRecord({ value: 10 });
    const output = renderer
      .create(<Counter counter={counterRecord} />)
      .toJSON();
    expect(output).toMatchSnapshot();
    ReactDOM.unmountComponentAtNode(div);
  });
});
