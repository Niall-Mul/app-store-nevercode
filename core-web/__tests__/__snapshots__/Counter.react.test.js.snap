// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter Component renders without crashing 1`] = `
<div
  style={
    Object {
      "paddingTop": "20px",
    }
  }
>
  <h2
    className="ui header"
  >
    <i
      aria-hidden="true"
      className="stopwatch icon"
    />
     
    10
  </h2>
  <button
    className="ui primary button"
    disabled={undefined}
    onClick={[Function]}
    role="button"
    tabIndex={undefined}
  >
    Increase
  </button>
</div>
`;
